kore/src/Kore/Step/Simplification/Not.hs

```diff
@@ -121,16 +123,16 @@ makeEvaluateNot
     => Not sort (Pattern variable)
     -> OrPattern variable
 makeEvaluateNot Not { notChild } =
-    OrPattern.fromPatterns
-        [ Pattern.fromTermLike $ TermLike.markSimplified $ makeTermNot term
-        , Pattern.fromConditionSorted
+    merge
```

Comment:
This looks a bit confusing at first sight, since `merge` is a fairly generic name. How about using it qualified (`OrPattern.merge`)? The same for `OrPattern.singleton`. I think that both are only used with qualifiers everywhere else.

Reply:
Done :+1: